.github/workflows/test.yml

```diff
@@ -1,33 +1,51 @@
 name: CI
 
 on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+    push:
```

Comment:
Is this file formatted by prettier / eslint? I would have expected no changes here

Reply:
we can control what files get changed. 
I believe it was prettier that changed this file

Reply:
Alright. I'm fine with just letting it do its thing